scalikejdbc-mapper-generator-core/src/main/scala/scalikejdbc/mapper/CodeGenerator.scala

```diff
@@ -854,6 +904,12 @@ class CodeGenerator(table: Table, specifiedClassName: Option[String] = None)(imp
           |      val shouldBeNone = %className%.find(%primaryKeys%)
           |      shouldBeNone.isDefined should beFalse
           |    }
+          |    "perform batch insert" in new AutoRollback {
+          |      val entities = %className%.findAll()
+          |      entities.foreach(e => %className%.destroy(e))
+          |      val batchInserted = %className%.batchInsert(entities.toList)
```

Comment:
If I remove toList, the below test case will fail..

https://github.com/scalikejdbc/scalikejdbc/blob/master/scalikejdbc-mapper-generator/src/sbt-test/scalikejdbc-mapper-generator/gen/test#L77